Directory.Build.props

```diff
@@ -59,6 +61,10 @@
     <!-- CoreFx sepcific paths -->
     <TestWorkingDir Condition="'$(TestWorkingDir)'==''">$(BinDir)tests/</TestWorkingDir>
 
+    <TestArchiveRoot>$(ProjectDir)artifacts/helix/</TestArchiveRoot>
+    <TestArchiveTestsDir>$(TestArchiveRoot)tests/</TestArchiveTestsDir>
+    <TestArchiveRuntimeDir>$(TestArchiveRoot)runtime/</TestArchiveRuntimeDir>
```

Comment:
Are these guys missing to make them configuration specific? i.e `<TestArchiveRuntimeDir>$(TestArchiveRoot)runtime/$(BuildConfiguration)</TestArchiveRuntimeDir>` ?

Reply:
Good catch. The TestArchiveRoot should use the arcade artifactsdir property and the TestArchiveTestsDir properties should contain the buildconfiguration. thanks.